src/Microsoft.VisualStudio.Threading.Analyzers/VSSDK008UseAwaitInAsyncMethodsCodeFix.cs

```diff
@@ -135,7 +135,7 @@ public override string Title
                 if (this.AlternativeAsyncMethod != string.Empty)
                 {
                     // Replace the member being called and await the invocation expression.
-                    var asyncMethodName = SyntaxFactory.IdentifierName(this.diagnostic.Properties[AsyncMethodKeyName]);
+                    var asyncMethodName = syncMethodName.WithIdentifier(SyntaxFactory.Identifier(this.diagnostic.Properties[AsyncMethodKeyName]));
```

Comment:
why has this line been changed? is it because of the change above where you now use invokedMethodName.ToString()